source/base/exceptions.cc

```diff
@@ -70,7 +70,12 @@ ExceptionBase::ExceptionBase ()
   stacktrace (NULL),
   n_stacktrace_frames (0),
   what_str("")
-{}
+{
+#ifdef DEAL_II_HAVE_GLIBC_STACKTRACE
+  for (unsigned int i=0; i<sizeof(raw_stacktrace)/sizeof(raw_stacktrace[0]); ++i)
+    raw_stacktrace[0] = NULL;
```

Comment:
Ah, I was too slow pushing my correction :-( It should of course have read `[i]`...